### ISSUE TITLE
AF2 Colabfold (using MMSeqs2) rapid crash with "Bio" module not found, due to BiopythonDeprecation "Warning"

### ISSUE THREAD

**Issue by jfbazan** [sokrypton/ColabFold#562]
This is a new issue that just arose today with ColabFold V1.5.5, just running a routine AF2 multimer job, and one gets a very quick crash as soon as the 'Run Prediction" module starts running, with the short error msg below related to "Bio" module not found because of Biopython deprecation (which may be tied to a bug a couple of weeks ago relating to something that broke when Biopython was upgraded, & the temporary fix--that worked!--was to deprecate biopython) which Google's Colab now finds fault with. Thx in advance for your expert help.

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-10-06098a2c69e9>](https://uyehnjbdqv-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab_20240125-060137_RC00_601416831#) in <cell line: 7>()
      5 import warnings
      6 warnings.simplefilter(action='ignore', category=FutureWarning)
----> 7 from Bio import BiopythonDeprecationWarning
      8 warnings.simplefilter(action='ignore', category=BiopythonDeprecationWarning)
      9 from pathlib import Path

ModuleNotFoundError: No module named 'Bio'

---------------------------------------------------------------------------

**Comment by sokrypton** [comment]
I was not able to reproduce the error. It worked with multimer model.

Can you try again? I wonder if there was an issue just finishing the first cell (didn't completely install).

Did you change any other default options?

**Comment by jfbazan** [comment]
Odd, no, didn’t change. When that happened, I rebooted the Colab and tried again, same fault. Let me try again, different browser with clean reboot…Sent from my iPhoneOn Jan 27, 2024, at 4:43 PM, Sergey O ***@***.***> wrote:﻿
I was not able to reproduce the error. It worked with multimer model.
Can you try again? I wonder if there was an issue just finishing the first cell (didn't completely install).
Did you change any other default options?

**Comment by jfbazan** [comment]
alright, I deleted all the old jobs, totally clean start, and now it runs fine. I have no idea why that earlier error happened, but thx so much for your quick response. Blaming an errant cosmic ray, have a good wkend!

**Comment by sokrypton** [comment]
To me this error is usually an indication that something failed to install. It's not finding the module at all, let alone it potentially being issues with version.
`ModuleNotFoundError: No module named 'Bio'`

blame it on pypi or github or internet connection at google :P
